Source/Charts/Data/Implementations/Standard/ChartData.swift

```diff
@@ -508,32 +324,22 @@ open class ChartData: NSObject, ExpressibleByArrayLiteral
     
     /// Enables / disables highlighting values for all DataSets this data object contains.
     /// If set to true, this means that values can be highlighted programmatically or by touch gesture.
-    @objc open var highlightEnabled: Bool
+    @objc open var isHighlightEnabled: Bool
     {
-        get
-        {
-            return first { $0.highlightEnabled == false } == nil
-        }
-        set
-        {
-            forEach { $0.highlightEnabled = newValue }
-        }
+        get { return first { $0.highlightEnabled == false } == nil }
+        set { forEach { $0.highlightEnabled = newValue } }
     }
-    
-    /// if true, value highlightning is enabled
-    @objc open var isHighlightEnabled: Bool { return highlightEnabled }
-    
+
     /// Clears this data object from all DataSets and removes all Entries.
     /// Don't forget to invalidate the chart after this.
     @objc open func clearValues()
     {
-        dataSets.removeAll(keepingCapacity: false)
-        notifyDataChanged()
+        removeAll(keepingCapacity: false)
```

Comment:
sorry I didn't see we write our own `removeAll`? And calling notifyDataChanged

Reply:
line 473

Reply:
what.. my xocde search must be mistaken at that time. I cannot find it. Now it could, sorry